irohad/main/application.cpp

```diff
@@ -203,6 +208,20 @@ bool Irohad::restoreWsv() {
       });
 }
 
+bool Irohad::updatePeers() {
+  // drop old peers and insert new ones if --peers flag is set
+  if (not initial_peers_.empty()) {
+    storage->resetPeers();
+    for (const auto &peer : initial_peers_) {
+      if (not storage->insertPeer(*peer)) {
+        log_->error("Peer insertion failed");
+        return false;
```

Comment:
Well, I think logs should be removed here since we already have more detailed logs in storage.
Returning value is an issue but I just used the same design as `restoreWsv`. Maybe we should add some checks in `init` function?